### PR TITLE
aircrack-ng: Add devel

### PR DESCRIPTION
**Pull request by equal-l2**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by equal-l2** [comment]
Hmm, airodump-ng-oui-update on devel seems not to work.
This will need a patch...

Is it impossible to put multiple patches in END block?

**Comment by equal-l2** [comment]
Well, I've noticed there is homebrew-patch.
I'll make a PR there for a patch.

File: Formula/aircrack-ng.rb [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-  url "https://download.aircrack-ng.org/aircrack-ng-1.1.tar.gz"
-  sha256 "b136b549b7d2a2751c21793100075ea43b28de9af4c1969508bb95bcc92224ad"
   revision 2
 
+  stable do
+    url "https://download.aircrack-ng.org/aircrack-ng-1.1.tar.gz"
+    sha256 "b136b549b7d2a2751c21793100075ea43b28de9af4c1969508bb95bcc92224ad"
+    # Remove root requirement from OUI update script. See:
+    # https://github.com/Homebrew/homebrew/pull/12755
+    patch :DATA
```
Comment (MikeMcQuaid) [inline Formula/aircrack-ng.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. I realise you didn't create it but if you could submit it that'd be great. Thanks!

File: Formula/aircrack-ng.rb [hunk 2]
```diff
@@ -16,19 +19,27 @@ class AircrackNg < Formula
...
   def install
-    # Fix incorrect OUI url
-    inreplace "scripts/airodump-ng-oui-update",
-      "http://standards.ieee.org/regauth/oui/oui.txt",
-      "http://standards-oui.ieee.org/oui.txt"
+    unless build.devel?
+      # Fix incorrect OUI url
+      inreplace "scripts/airodump-ng-oui-update",
+        "http://standards.ieee.org/regauth/oui/oui.txt",
+        "http://standards-oui.ieee.org/oui.txt"
```
Comment (MikeMcQuaid) [inline Formula/aircrack-ng.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. I realise you didn't create it but if you could submit it that'd be great. Thanks!

File: Formula/aircrack-ng.rb [hunk 3]
```diff
@@ -16,19 +19,27 @@ class AircrackNg < Formula
     sha256 "02efed81e48c8f70bbd1d3051e84b25815fcceb7166cb79d472f9552a4708ae2" => :mavericks
   end
 
+  devel do
+    url "https://download.aircrack-ng.org/aircrack-ng-1.2-rc4.tar.gz"
+    sha256 "d93ac16aade5b4d37ab8cdf6ce4b855835096ccf83deb65ffdeff6d666eaff36"
+    patch do
+      # The url will be replaced with one on formula-patch once PR on formula-patch has merged
+      url "https://github.com/equal-l2/formula-patches/raw/aircrack-ng/aircrack-ng/aircrack-ng-devel.patch"
```
Comment (MikeMcQuaid) [inline Formula/aircrack-ng.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by nijikon** [comment]
@equal-l2 ping

**Pull request closed without merging**